webapp/src/components/data_selectors/jira_autocomplete_selector/jira_autocomplete_selector.tsx

```diff
@@ -44,6 +44,8 @@ export default class JiraAutoCompleteSelector extends React.PureComponent<Props>
                 value: suggestion.value,
                 label: stripHTML(suggestion.displayName),
             }));
+        }).catch((e) => {
+            throw new Error('Error fetching data');
```

Comment:
We are sort of "swallowing" this error here. Is the error present in `e` helpful to the user at all?

Reply:
I guess I'm wondering why we're returning an error from the API if we're not using it on the frontend though

Reply:
@mickmister We want to show the error to the user in case we are unable to fetch the data. If we don't send the error from the API we won't be able to display the generic error. Please correct me if I am wrong

Reply:
Alright that makes sense